commons/commons-util/src/main/kotlin/ebi/ac/uk/dsl/excel/ExcelSheet.kt

```diff
@@ -1,13 +1,17 @@
 package ebi.ac.uk.dsl.excel
 
+import ebi.ac.uk.base.EMPTY
 import org.apache.poi.xssf.streaming.SXSSFSheet
 
 class ExcelSheet(private val sheet: SXSSFSheet) {
     private val rowIterator = buildIterator()
 
     fun row(cellsBuilder: ExcelRow.() -> Unit) = ExcelRow(newRow()).apply { cellsBuilder() }
 
-    fun emptyRow() = row { }
+    fun emptyRow() = row {
```

Comment:
this is only used as part of test/itest , IMO it should be in  test class package or function

Reply:
All DSLs we have are used for testing, perhaps we could create a separate module in another PR and move them all? Right now they're in `commons-util` but we could have something like `commons-dsl` ?